packages/custom_lint/test/cli_test.dart

```diff
@@ -228,43 +328,6 @@ Analyzing...
     );
   });
 
-  test('CLI lists warnings from all plugins and set exit code', () async {
```

Comment:
This test is a duplicate

Reply:
Do you have the source of the other one?

Reply:
The one with the JSON above https://github.com/invertase/dart_custom_lint/pull/198/files#diff-0ec183ab5d11bde0260ed0f0d4a24da1edcd2c8bd6d3fb88f94566298721358fL70